Gemfile

```diff
@@ -0,0 +1,77 @@
+source 'https://rubygems.org'
+
+ruby '3.2.2'
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem 'rails', '~> 7.1.1'
+
+# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
+gem 'sprockets-rails'
+
+# Use postgresql as the database for Active Record
+gem 'pg'
+
+# Add rubocop linter
+gem 'rubocop', '>= 1.0', '< 2.0'
+
+# Use the Puma web server [https://github.com/puma/puma]
+gem 'puma', '>= 5.0'
+
+# Use JavaScript with ESM import maps [https://github.com/rails/importmap-rails]
+gem 'importmap-rails'
+
+# Hotwire's SPA-like page accelerator [https://turbo.hotwired.dev]
+gem 'turbo-rails'
+
+# Hotwire's modest JavaScript framework [https://stimulus.hotwired.dev]
+gem 'stimulus-rails'
+
+# Build JSON APIs with ease [https://github.com/rails/jbuilder]
+gem 'jbuilder'
+
+# Use Redis adapter to run Action Cable in production
+# gem "redis", ">= 4.0.1"
+
+# Use Kredis to get higher-level data types in Redis [https://github.com/rails/kredis]
+# gem "kredis"
+
+# Use Active Model has_secure_password [https://guides.rubyonrails.org/active_model_basics.html#securepassword]
+# gem "bcrypt", "~> 3.1.7"
+
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'tzinfo-data', platforms: %i[windows jruby]
+
+# Reduces boot times through caching; required in config/boot.rb
+gem 'bootsnap', require: false
+
+# Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
+# gem "image_processing", "~> 1.2"
+
+# Use devise for authentication
+gem 'devise'
+
+group :development, :test do
+  # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
+  gem 'debug', platforms: %i[mri windows]
+  gem 'rails-controller-testing'
+  gem 'rspec-rails'
```

Comment:
![image](https://github.com/henokkhm/microverse-ror-group-capstone/assets/93039359/c8e5f15a-68fa-4a69-b77a-0a1de1630d35)

- [ ] Great work so far but it would be nice if you make your tests pass because tests tell us if the app is working as expected or not so it would be very great if you fix all the tests that are failing :+1:

Reply:
Thanks, @HAMMAS-SALEEM for bringing this to our notice. However, all tests happen to pass on my device and my partner's devices as seen in the attachment. The screenshot doesn't contain the details of the failing tests which would have been helpful. Can you kindly check to see that the issue isn't from your machine?

<img width="791" alt="image" src="https://github.com/henokkhm/microverse-ror-group-capstone/assets/99335288/3fe1605a-c0a5-45d5-921b-10afb580724f">